src/desktop/ipc/RemoteBridge.ts

```diff
@@ -64,6 +64,9 @@ export class RemoteBridge {
 	}
 
 	destroyBridge(window: ApplicationWindow) {
-		primaryIpcHandler.removeHandler(window._browserWindow.webContents.id)
+		if(!window._browserWindow || window._browserWindow.isDestroyed()) return
+		const webContents = window._browserWindow.webContents
+		if(!webContents || webContents.isDestroyed()) return
```

Comment:
so if the window is destroyed then we don't ever remove the handler? Or is it removed from elsewhere? Maybe worth a comment.

Also formatting.